### ISSUE TITLE
Expose more configuration options at component API level

### ISSUE THREAD

**Issue by MortenHofft** [gbif/hosted-portals#84]
The widgets can already be configured to 

* show certain table columns
* custom map styles
* ~default visible filters buttons in the filter bar~
* ~available filters~
* ~configure tabs~

But it isn't exposed and documented at the component level

Additional settings for which tabs to show etc is of course also valuable, but we should start by exposing the work already implemented.

**Comment by tucotuco** [comment]
Where can the column and map customizations be made?

**Comment by MortenHofft** [comment]
@tucotuco Neither it isn't exposed at the component level so you cannot set it - that is what this issue is a task to do.

The ambition is to expose config options for maps, but since maps probably needs redoing I have postponed it. I suspect we will leave Mapbox and go back to Openlayers. Mapbox makes beautiful and fast maps, but they only support mercator projection and we need to show data near the poles. The tile server currently generating the map tiles is one I made as an experiment years ago. And it only supports mercator. And I'm honestly not sure how to change that, so I suspect that will take some time.

The issue for map projections https://github.com/gbif/hosted-portals/issues/69
